new/cookbook-segment-deprecation.md

```diff
@@ -0,0 +1,51 @@
+---
+RFC: unassigned
+Author: Lamont Granquist <lamont@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Cookbook Segment Deprecation
+
+## Motivation
+
+    As a Chef User/Developer,
+    I want to be able to extend the structure of cookbooks without fighting with a wire protocol
+    written 6 years ago,
```

Comment:
I lol'd but I'm not sure this is the best motivation description :).  How about,

> As a Chef User/Developer
> I want to be able to extend the structure of cookbooks easily,
> in order to quickly adapt to new automation needs.